### ISSUE TITLE
bug: Failing builds with the latest Revanced CLI

### ISSUE THREAD

**Issue by SinTan1729** [revanced/revanced-cli#175]
### Type

Error while running the CLI

### Bug description

While trying to apply the patches, I get this error.

### Steps to reproduce

1. Download the latest Revanced packages.
2. Try to apply patches.

### Relevant log output

```shell
Exception in thread "main" kotlin.jvm.KotlinReflectionNotSupportedError: Kotlin reflection implementation is not found at runtime. Make sure you have kotlin-reflect.jar in the classpath
        at kotlin.jvm.internal.ClassReference.error(ClassReference.kt:88)
        at kotlin.jvm.internal.ClassReference.getNestedClasses(ClassReference.kt:24)
        at kotlin.reflect.full.KClasses.getCompanionObject(KClasses.kt:47)
        at app.revanced.patcher.extensions.PatchExtensions.getOptions(AnnotationExtensions.kt:62)
        at app.revanced.utils.OptionsLoader.generateDefaults(OptionsLoader.kt:68)
        at app.revanced.utils.OptionsLoader.init(OptionsLoader.kt:25)
        at app.revanced.cli.command.MainCommand.run(MainCommand.kt:137)
        at picocli.CommandLine.executeUserObject(CommandLine.java:2026)
        at picocli.CommandLine.access$1500(CommandLine.java:148)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
        at picocli.CommandLine.execute(CommandLine.java:2170)
        at app.revanced.cli.main.MainKt.main(Main.kt:7)
```

### Additional context

I'm using the build command `java -jar revanced-cli.jar -m revanced-integrations.apk -b revanced-patches.jar -a com.google.android.youtube.apk -o revanced-yt-nonroot.apk`.
The output of `java --version` is:
```
openjdk 17.0.5 2022-10-18 LTS
OpenJDK Runtime Environment Zulu17.38+21-CA (build 17.0.5+8-LTS)
OpenJDK 64-Bit Server VM Zulu17.38+21-CA (build 17.0.5+8-LTS, mixed mode, sharing)
```

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by SinTan1729** [comment]
With the latest update of `revanced-cli (v2.18.1)`, I'm getting a different error.
```
INFO: Reading dex files
INFO: Decoding AndroidManifest.xml only, because resources are not needed
INFO: Merging revanced-integrations.apk
INFO: Skipping theme: Excluded by default
INFO: Deleting existing resource cache directory
INFO: Decoding resources
INFO: always-autorepeat succeeded
INFO: client-spoof succeeded
INFO: comments succeeded
INFO: custom-branding succeeded
INFO: custom-video-buffer succeeded
INFO: custom-video-speed succeeded
INFO: debugging succeeded
INFO: disable-auto-captions succeeded
INFO: disable-auto-player-popup-panels succeeded
INFO: disable-fullscreen-panels succeeded
INFO: disable-startup-shorts-player succeeded
INFO: disable-zoom-haptics succeeded
INFO: downloads succeeded
Exception in thread "main" java.lang.NoClassDefFoundError: app/revanced/patcher/data/ContextKt
        at app.revanced.patches.youtube.layout.widesearchbar.patch.WideSearchbarPatch$Companion.walkMutable(WideSearchbarPatch.kt:48)
        at app.revanced.patches.youtube.layout.widesearchbar.patch.WideSearchbarPatch.execute(WideSearchbarPatch.kt:89)
        at app.revanced.patches.youtube.layout.widesearchbar.patch.WideSearchbarPatch.execute(WideSearchbarPatch.kt:27)
        at app.revanced.patcher.Patcher.executePatches$executePatch(Patcher.kt:425)
        at app.revanced.patcher.Patcher.access$executePatches$executePatch(Patcher.kt:44)
        at app.revanced.patcher.Patcher$executePatches$1.invokeSuspend(Patcher.kt:445)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlin.sequences.SequenceBuilderIterator.hasNext(SequenceBuilder.kt:127)
        at app.revanced.utils.patcher.PatcherKt.applyPatchesVerbose(Patcher.kt:83)
        at app.revanced.cli.patcher.Patcher.start$revanced_cli(Patcher.kt:20)
        at app.revanced.cli.command.MainCommand.run(MainCommand.kt:155)
        at picocli.CommandLine.executeUserObject(CommandLine.java:2026)
        at picocli.CommandLine.access$1500(CommandLine.java:148)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
        at picocli.CommandLine.execute(CommandLine.java:2170)
        at app.revanced.cli.main.MainKt.main(Main.kt:7)
Caused by: java.lang.ClassNotFoundException: app.revanced.patcher.data.ContextKt
        at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:445)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:587)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
        ... 20 more
```

**Comment by EnolaGae** [comment]
Glad I'm not the only one with this issue... I spent 4 hours troubleshooting only to find out it came from here 🐗

**Comment by PatrykMis** [comment]
Fixed in [v2.18.2-dev.1](https://github.com/revanced/revanced-cli/releases/tag/v2.18.2-dev.1).
